### PR TITLE
Update 3.4/get-pip.py to latest

### PR DESCRIPTION
**Pull request by CyanDevs**
get-pip.py was updated here: https://github.com/pypa/get-pip/commit/fa88b51870e19de759a4615937b02ff334741758#diff-72d73db944cf6d9a5f11d6c073c1dce0bde28a4d2341caf4ae976e858c62cabc and was released February 21, 2021.

Signed-off-by: Chris Yan <chrisyan@microsoft.com>

### CODE REVIEW

**Comment by CyanDevs** [comment]
bors try

File: scripts/install-windows-prereqs.ps1 [hunk 1]
```diff
@@ -33,7 +33,7 @@ Param(
     [string]$NSISURL = 'https://oejenkins.blob.core.windows.net/oejenkins/nsis-3.05-setup.exe',
     [string]$NSISHash = '1A3CC9401667547B9B9327A177B13485F7C59C2303D4B6183E7BC9E6C8D6BFDB',
     [string]$GetPipURL = 'https://bootstrap.pypa.io/3.4/get-pip.py',
-    [string]$GetPipHash = '564FABC2FBABD9085A71F4A5E43DBF06D5CCEA9AB833E260F30EE38E8CE63A69',
+    [string]$GetPipHash = 'DBD5DAE3D1E7F6DF844D630CDF65E0F0D98E483C9997DAEA17C7C9D86F7B38AD',
```
Comment (mingweishih) [inline scripts/install-windows-prereqs.ps1:36]: To clarify: we still don't have e2e test on windows that tests this script, right?

**Review by anakrish — APPROVED** [review]

**Review by radhikaj — APPROVED** [review]
Bors r+

**Comment by radhikaj** [comment]
Bors delegate=cyandevs

File: scripts/install-windows-prereqs.ps1 (hunk 1 above)
Comment (CyanDevs) [reply scripts/install-windows-prereqs.ps1:36]: We do somewhere, I think. @rs-- may be able to give a more precise answer.
Meanwhile, I have been testing this install script using the same pipeline to update the Windows images.

**Pull request merged**